data/hails.txt

```diff
@@ -1098,6 +1169,11 @@ phrase "friendly navy"
 		"the biggest"
 		"the largest"
 		"the greatest"
+		"the strongest"
```

Comment:
"Join the Navy, the strongest humanitarian organization in the galaxy!"

Would anyone describe a humanitarian organization as "strong?"

Reply:
I would. The US is humanitarian but it's pretty strong.

Reply:
Well to describe the Navy as the strongest humanitarian organization in the galaxy is probably entirely correct. They do have the biggest guns in human space.